win32/build/confutils.js

```diff
@@ -94,10 +94,10 @@ if (typeof(CWD) == "undefined") {
 
 /* defaults; we pick up the precise versions from configure.ac */
 var PHP_VERSION = 8;
-var PHP_MINOR_VERSION = 0;
+var PHP_MINOR_VERSION = 1;
 var PHP_RELEASE_VERSION = 0;
 var PHP_EXTRA_VERSION = "";
-var PHP_VERSION_STRING = "8.0.0";
+var PHP_VERSION_STRING = "8.1.0";
```

Comment:
@cmb69 Let me know if there's something else that needs to be updated on the Windows side :)

Reply:
Nope, that's it. :)